src/Microsoft.Dnx.Runtime.Internals/RuntimeEnvironmentExtensions.cs

```diff
@@ -31,8 +32,64 @@ public static string GetShortVersion(this IRuntimeEnvironment env)
             return $"{env.RuntimeType}-{env.RuntimeArchitecture}-{env.RuntimeVersion}";
         }
 
+        public static IEnumerable<string> GetDefaultRestoreRuntimes(this IRuntimeEnvironment env)
```

Comment:
Sure. I think I just put it here for convenience and because the runtime is the one that actually defines it.